docs/processes/release/tasks/release-build.md

```diff
@@ -0,0 +1,261 @@
+> (e.g. release-61-1, release-61-2) for building release files.
+> 2. Run `$ ant releaseVer` once to make sure there are no build configuration
+> issues. Run `$ ant clean` to clean up build.
+> 3. \[Optional\] In the next step, the build target invokes gpg to generate
+> signature of build output files. gpg will request you to type in the
+> passphrase for the ICU Project (icuintl@us.ibm.com) PGP key once. gpg pinentry
+> is used for this purpose, but it may not work well on some systems when pgp
+> was invoked from ant (such as Mac OS X). If this is the case, you would want
+> to run gpg-agent to cache passphrase. More specifically, once you set up
+> gpg-agent, run` $ gpg -ab -u "ICU Project" foo.txt` to sign arbitrary file.
+> This operation will prompt you to type in the ICU Project key's passphrase and
+> cache it. So it does not need to ask you to type passphrase again while
+> running the ant release target.
+> 4. Run `$ ant releaseFinal` . This target generates release files for ICU
+> site, as well as files for Maven Central Repository, then upload the files to
+> OSSRH staging repository. You should see build log as below at the end.
+> > ` [deploy] Transferring 37K from icu4j-releases`
+
+> > ` [deploy] Uploaded 37K`
+
+> > ` [deploy] Uploading:
+> > com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-sources.jar.asc to
+> > repository icu4j-releases at
+> > https://oss.sonatype.org/service/local/staging/deploy/maven2`
+
+> > ` [deploy] Transferring 0K from icu4j-releases`
+
+> > ` [deploy] Uploaded 0K`
+
+> > ` [deploy] Uploading:
+> > com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-javadoc.jar to
+> > repository icu4j-releases at
+> > https://oss.sonatype.org/service/local/staging/deploy/maven2`
+
+> > ` [deploy] Transferring 22K from icu4j-releases`
+
+> > ` [deploy] Uploaded 22K`
+
+> > ` [deploy] Uploading:
+> > com/ibm/icu/icu4j-localespi/61.1/icu4j-localespi-61.1-javadoc.jar.asc to
+> > repository icu4j-releases at
+> > https://oss.sonatype.org/service/local/staging/deploy/maven2`
+
+> > ` [deploy] Transferring 0K from icu4j-releases`
+
+> > ` [deploy] Uploaded 0K`
+
+> > `BUILD SUCCESSFUL`
+
+> > `Total time: 6 minutes 19 seconds`
+
+> After the successful target execution, copy files generated under `release`
+> directory and upload them to ICU site.
+> 5. Log on to [OSSRH Nexus Repository Manager](https://oss.sonatype.org/).
+> Select "Staging Repositories" under "Build Promotion" on the left navigation
+> pane, then type "com.ibm.icu" on the search field on the upper right corner.
+> You should find a repository entry corresponding to the one just uploaded.
+> Select the entry, and sanity check content. Once you confirm the contents,
+> click "Close" on the menu. If the repository is closed successfully, you can
+> "Release" after that. The release operation will promote the staging
+> repository to production. It usually takes 20 to 30 minutes.
+
+> ![image](../../../setup/java/ant/Capture.PNG)
```

Comment:
This image does not render.

Reply:
I added the png as NexusRepoMan.png, and I'm not touching setup/java/ant/ yet, so I added it in the release/tasks/ directory.